src/gui/MainWindow.cpp

```diff
@@ -135,7 +135,7 @@ MainWindow::MainWindow() :
 		embed::getIconPixmap("home").transformed(QTransform().rotate(90)), splitter, false));
 
 	QStringList root_paths;
-	QString title = tr( "Root directory" );
+	QString title = tr( "Root Directory" );
```

Comment:
```suggestion
	QString title = tr("Root Directory");
```
While you are at it, consider fixing up the style too.